src/main/java/uk/gov/justice/digital/delius/service/AlfrescoService.java

```diff
@@ -56,21 +53,11 @@ public SearchResult listDocuments(String crn) {
     }
 
     public ResponseEntity<Resource> getDocument(String documentId, String crn) {
+        val maybeDocumentMeta = getDocumentDetail(documentId, crn);
 
-        // Check crn exists
```

Comment:
Previous this was retrieving *all* documents for an offender and finding the one it wants rather than simply asking Alfresco for the specific document (and then comparing crn)